CMake/Dependencies/libkvsCommonLws-CMakeLists.txt

```diff
@@ -4,21 +4,43 @@ project(libkvsCommonLws-download NONE)
 
 include(ExternalProject)
 
-ExternalProject_Add(libkvsCommonLws-download
-    GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-producer-c.git
-    GIT_TAG           develop
-    PREFIX            ${CMAKE_CURRENT_BINARY_DIR}/build
-    CMAKE_ARGS        
-      -DCMAKE_INSTALL_PREFIX=${OPEN_SRC_INSTALL_PREFIX}
-      -DBUILD_COMMON_LWS=ON
-      -DBUILD_COMMON_CURL=OFF
-      -DBUILD_DEPENDENCIES=FALSE
-      -DOPEN_SRC_INSTALL_PREFIX=${OPEN_SRC_INSTALL_PREFIX}
-      -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
-      -DUSE_OPENSSL=${USE_OPENSSL}
-      -DUSE_MBEDTLS=${USE_MBEDTLS}
-      -DKVS_DEFAULT_STACK_SIZE=${KVS_DEFAULT_STACK_SIZE}
-      -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
-    BUILD_ALWAYS      TRUE
-    TEST_COMMAND      ""
-)
+if(WIN32)
+  ExternalProject_Add(libkvsCommonLws-download
+      GIT_REPOSITORY    https://github.com/awslabs/amazon-kinesis-video-streams-producer-c.git
+      GIT_TAG           develop
+      PREFIX            ${CMAKE_CURRENT_BINARY_DIR}/build
+      CMAKE_ARGS        
+        -DCMAKE_INSTALL_PREFIX=${OPEN_SRC_INSTALL_PREFIX}
+        -DBUILD_COMMON_LWS=ON
+        -DBUILD_COMMON_CURL=OFF
+        -DBUILD_DEPENDENCIES=FALSE
+        -DOPEN_SRC_INSTALL_PREFIX=${OPEN_SRC_INSTALL_PREFIX}
+        -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+        -DUSE_OPENSSL=${USE_OPENSSL}
+        -DUSE_MBEDTLS=${USE_MBEDTLS}
+        -DKVS_DEFAULT_STACK_SIZE=${KVS_DEFAULT_STACK_SIZE}
+        -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
+        -DPKG_CONFIG_EXECUTABLE=${PKG_CONFIG_EXECUTABLE}
```

Comment:
Can we pass `PKG_CONFIG_EXECUTABLE` to Producer C in the non-Windows case too? If not, can you please define `CMAKE_ARGS` above `ExternalProject_Add` and set like this to avoid clutter:

```
set(CMAKE_ARGS
        -DCMAKE_INSTALL_PREFIX=${OPEN_SRC_INSTALL_PREFIX}
        -DBUILD_COMMON_LWS=ON
        ...)

if (WIN32)
    set(CMAKE_ARGS ${CMAKE_ARGS} -DPKG_CONFIG_EXECUTABLE=...)
endif()

ExternalProject_Add(project_libcurl
    CMAKE_ARGS        ${CMAKE_ARGS}
     ...)
```